### ISSUE TITLE
Neo field fails to load when viewing a revision

### ISSUE THREAD

**Issue by N0ps32** [spicywebau/craft-neo#274]
#### Description
I think I encountered a bug in Neo. When trying to access a revision of a page with a large Neo block in it it fails to load:

```
jquery.js:3850 Uncaught TypeError: Cannot read property 'removeAttribute' of undefined
    at s.constructor.init (main.js?v=1571319030:13)
    at s.constructor.constructor (garnish.js:768)
    at s.constructor (garnish.js:40)
    at new s.constructor (garnish.js:40)
    at Object.createInput (main.js?v=1571319030:1)
    at HTMLDocument.<anonymous> (50-studium-executive-education?site=default&revisionId=5607:759)
    at e (jquery.js:3557)
    at t (jquery.js:3625) 
```

This line seems to cause the issue:
```
this.$confirmUnloadForms[0].removeAttribute("data-confirm-unload")
```

#### Steps to reproduce
1. Open entry
2. Open revision dropdown and select a revision

#### Other information
- Neo version: Neo 2.5.5
- Craft version: Craft Pro 3.3.11
- (if multi-site) Are the affected Neo fields set to manage blocks on a per-site basis: Yes
- (if multi-site, > 2.4.0) What is the propagation method of the neo field: Save blocks to all sites the owner element is saved in
- Is eager-loading used? Not sure

**Comment by pvldigital** [comment]
Hi @N0ps32,

A fix has been released for this issue in v2.5.6.

**Comment by N0ps32** [comment]
Thank you! That seems to have fixed the issue
